### ISSUE TITLE
Caller to 'demux' list of multiple errors returned

### ISSUE THREAD

**Issue by Vasilis** [uber-go/multierr#10]
If the caller is optionally aware that a multierr might be returned, she can access the list of errors and switch case among them. As a result, she is now able to act upon them accordingly.

**Comment by abhinav** [comment]
Agreed that this makes sense to add. We hope to add it soon. See also #6.

**Comment by Vasilis** [comment]
Thanks, @abhinav

**Comment by abhinav** [comment]
Released in v1.1.0
